tests/parsers/bashparser/resources/bash_files/valid_github_action_bash.sh

```diff
@@ -11,3 +11,4 @@ echo "hash=${{ ${ FOO } }}"
 echo "hash=${{ $FOO  }}"
 echo "hash=${{ {FOO}  }}"
```

Comment:
Because we are not using greedy regex matching anymore for the GitHub Action expression, do we want to remove those tests as they are not relevant anymore?

Reply:
After thinking more about it, I don't think it makes sense to remove them given that it ensures the issue of greedy matching not happening again. I will resolve this conversation.